Index: app/src/main/java/com/geridea/trentastico/logger/BugLogger.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/geridea/trentastico/logger/BugLogger.java	(revision 027904dbdc402b1572f4d645435508a877d52c2e)
+++ app/src/main/java/com/geridea/trentastico/logger/BugLogger.java	(revision )
@@ -13,7 +13,17 @@
 import com.geridea.trentastico.utils.AppPreferences;
 
 import org.acra.ACRA;
-import org.acra.ErrorReporter;
+import org.acra.ErrorReporter;ad
+sa das
+d sa
+d sa
+ dsa
+d sa
+dsaa
+da
+sd
+asd
+asd
 
 public class BugLogger {
 
